data/ignition/files/usr/local/share/assisted-service/assisted-service.env

```diff
@@ -0,0 +1,18 @@
+AUTH_TYPE=none
+DB_HOST=127.0.0.1
+DB_NAME=installer
+DB_PASS=admin
+DB_PORT=5432
+DB_USER=admin
+DEPLOY_TARGET=onprem
+DISK_ENCRYPTION_SUPPORT=true
+DUMMY_IGNITION=false
+ENABLE_SINGLE_NODE_DNSMASQ=true
+HW_VALIDATOR_REQUIREMENTS=[{"version":"default","master":{"cpu_cores":4,"ram_mib":16384,"disk_size_gb":120,"installation_disk_speed_threshold_ms":10,"network_latency_threshold_ms":100,"packet_loss_percentage":0},"worker":{"cpu_cores":2,"ram_mib":8192,"disk_size_gb":120,"installation_disk_speed_threshold_ms":10,"network_latency_threshold_ms":1000,"packet_loss_percentage":10},"sno":{"cpu_cores":8,"ram_mib":16384,"disk_size_gb":120,"installation_disk_speed_threshold_ms":10}}]
+IMAGE_SERVICE_BASE_URL=http://{{.NodeZeroIP}}:8888
+IPV6_SUPPORT=true
+NTP_DEFAULT_SERVER=
+PUBLIC_CONTAINER_REGISTRIES=quay.io
+RELEASE_IMAGES=[{"openshift_version":"4.10","cpu_architecture":"x86_64","url":"quay.io/openshift-release-dev/ocp-release:4.10.0-rc.1-x86_64","version":"4.10.0-rc.1"}]
```

Comment:
Do we have any planned task to remove the hard-coded release image version?

Reply:
It'll be part of integrating with the installer, since the installer knows what version it is.